apps/cut_plane_detection

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# Copyright (c) 2015, Ecole Polytechnique Federale de Lausanne, Blue Brain Project
+# All rights reserved.
+#
+# This file is part of NeuroM <https://github.com/BlueBrain/NeuroM>
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#     1. Redistributions of source code must retain the above copyright
+#        notice, this list of conditions and the following disclaimer.
+#     2. Redistributions in binary form must reproduce the above copyright
+#        notice, this list of conditions and the following disclaimer in the
+#        documentation and/or other materials provided with the distribution.
+#     3. Neither the name of the copyright holder nor the names of
+#        its contributors may be used to endorse or promote products
+#        derived from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY
+# DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+'''Find and the cut plane for the given neuron(s)'''
+
+import argparse
+import json
+import logging
+import sys
+from tqdm import tqdm
+
+from neurom import load_neuron
+from neurom.apps.cut_plane_detection import find_cut_plane
+from neurom.io.utils import get_files_by_path
+from neurom.utils import NeuromJSON
+
+
+L = logging.getLogger(__name__)
+
+
+def get_parser():
+    '''Parse command line arguments'''
+    description = ('Cut plane detection: '
+                   ' finding the position of the cut plane (X, Y or Z). '
+                   'It returns the cut plane and the positions of all cut terminations.')
+
+    parser = argparse.ArgumentParser(description=description)
+
+    parser.add_argument('datapath', nargs='+', type=str,
+                        help='Path to a morphology data file or a directory')
+
+    parser.add_argument('-o', '--output', dest='output_file',
+                        help=('Output json file name'))
+
+    parser.add_argument('-w', '--width', type=int, default=3,
```

Comment:
Is the default 3 cherry-picked here ?

Reply:
I tried values between 1 and 20 and it was the one giving best results on the 40 thalamus morphologies

Reply:
maybe say what the unit is, and also put that as a comment, so people know where it came from